packages/mobile/src/screens/search-results-screen/tabs/TracksTab.tsx

```diff
@@ -25,13 +24,6 @@ const getSearchTracksLineupMetadatas = makeGetLineupMetadatas(
 export const TracksTab = () => {
   const lineup = useSelector(getSearchTracksLineupMetadatas)
   const dispatch = useDispatch()
-  useFocusEffect(
-    useCallback(() => {
-      return () => {
-        dispatch(tracksActions.reset())
```

Comment:
so we want to call reset?

Reply:
We don't need to call it here because otherwise it'll clear the lineup even if you're moving between tabs (we want to keep it in case you come back). Tested thoroughly and there doesn't seem to be any issues with leaving this out altogether.